integration-libs/opf/base/components/opf-quick-buy/apple-pay/apple-pay.service.ts

```diff
@@ -375,6 +379,35 @@ export class ApplePayService {
       );
   }
 
+  protected loadPdpOriginalCart(orderSuccess = false): void {
```

Comment:
I'm steering clear of using the name "Pdp." In GooglePay and ApplePay services, it was referred to as `SingleProduct`, and the context also resembles `OpfQuickBuyLocation.PRODUCT`.

Perhaps we could consider renaming it to `loadCartAfterSingleProductTransaction`?